quantum.config.js

```diff
@@ -115,6 +116,11 @@ module.exports = {
       dest: 'resources',
       watch: false
     },
+    {
+      files: 'content/guide/**/*.html',
+      base: 'content',
+      watch: false
```

Comment:
Include the `guide` html so we can have iframe examples easily (without having to go through quantum to generate the HTML)